packages/wonder-stuff-ci/src/exec-prom.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * A simple promisified version of child_process.exec, so we can `await` it
+ */
+import {exec, ExecOptions} from "child_process";
+
+export const bufferToString = (input: Buffer | string): string => {
+    if (typeof input === "string") {
+        return input;
+    } else {
+        return input.toString("utf8");
+    }
```

Comment:
**suggestion (non-blocking):** The following would be a bit more succinct:
```suggestion
    return (typeof input === "string") ? input : input.toString("utf8");
```

Reply:
Agree!